core/src/test/java/io/neow3j/protocol/core/methods/response/StackItemTest.java

```diff
@@ -115,6 +115,7 @@ public void testDeserializeIntegerStackItem() throws IOException {
         IntegerStackItem item = rawItem.asInteger();
         assertEquals(new BigInteger("1124"), item.getValue());
 
+        // TODO: 23.06.20 Michael: set empty content to default value - here to BigInteger.ZERO
```

Comment:
yes, this is important, IMHO!

Reply:
Actually, I've already implemented this with the following setter in `IntegerStackItem.java`. I'll remove the todo.

```java
    @JsonSetter("value")
    public void setValue(BigInteger value) {
        if (value != null) {
            this.value = value;
        } else {
            this.value = BigInteger.ZERO;
        }
    }
```